client/src/components/HistoricElementView/tests/HistoricElementView-test.js

```diff
@@ -1,54 +1,41 @@
-/* global jest, describe, beforeEach, it, expect */
+/* global jest, test, describe, beforeEach, it, expect */
 
 import React from 'react';
-import ReactTestUtils from 'react-dom/test-utils';
 import ElementalAreaHistoryFactory from '../HistoricElementView';
+import { render } from '@testing-library/react';
 
-describe('HistoricElementView', () => {
-  class FieldGroupStub extends React.Component {
-    getLegend() {
-      return 'nah';
-    }
-    getClassName() {
-      return 'ok';
-    }
-    render() {
-      return <div>Group</div>;
-    }
-  }
-  const datumSet = {
-    ElementType: 'Stub',
-    ElementTitle: 'Pretend Element',
-    ElementEditLink: 'http://localhost:8080/'
+function makeProps(obj = {}) {
+  return {
+    data: {
+      ElementID: 1,
+      ElementType: 'Stub',
+      ElementTitle: 'Pretend Element',
+      ElementEditLink: 'http://localhost:8080/',
+      tag: 'div'
+    },
+    ...obj
   };
-  const HistoricElementView = ElementalAreaHistoryFactory(FieldGroupStub);
-  let history = null;
-
-  // Temporarily disable (with xdescribe):
-  xdescribe('render', () => {
-    beforeEach(() => {
-      history = ReactTestUtils.renderIntoDocument(
-        <HistoricElementView data={datumSet} />
-      );
-    });
+}
 
-    it('The header should feature the Element type', () => {
-      const type = ReactTestUtils.findRenderedDOMComponentWithTag(history, 'small').textContent;
-      expect(type).toBeTruthy();
-      expect(type).toEqual('Stub');
-    });
+class FieldGroupStub extends React.Component {
+  getLegend() {
+    return 'nah';
+  }
+  getClassName() {
+    return 'ok';
+  }
+  render() {
+    return <div className="test-field-group-stub">Group</div>;
+  }
+}
 
-    it('should should list the Element\'s title', () => {
-      const title = ReactTestUtils.findRenderedDOMComponentWithClass(history, 'elemental-preview__detail').textContent;
-      expect(title).toBeTruthy();
-      expect(title).toMatch(/^Pretend Element\b/);
-    });
+const HistoricElementView = ElementalAreaHistoryFactory(FieldGroupStub);
 
-    it('should should have a link in the header', () => {
-      const link = ReactTestUtils.findRenderedDOMComponentWithClass(history, 'elemental-preview__link');
-      expect(link).toBeTruthy();
-      expect(link.href).toBeTruthy();
-      expect(link.textContent).toEqual('Block history');
-    });
-  });
+test('HistoricElementView render ', () => {
```

Comment:
The space at the end there makes me wonder if you had meant to do some more work on this one (either come up with a more descriptive title or split this back out into the individual well-described tests from the original)?
If not, then this is fine.

Reply:
Not doing